financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetState.kt

```diff
@@ -9,38 +11,22 @@ import com.stripe.android.financialconnections.model.FinancialConnectionsSession
  *  Class containing all of the data needed to represent the screen.
  */
 internal data class FinancialConnectionsSheetState(
+    val initialArgs: FinancialConnectionsSheetActivityArgs,
     val activityRecreated: Boolean = false,
-    val manifest: FinancialConnectionsSessionManifest? = null,
-    val authFlowActive: Boolean = false
-) {
+    @PersistState val manifest: FinancialConnectionsSessionManifest? = null,
+    @PersistState val authFlowActive: Boolean = false,
+    val viewEffect: FinancialConnectionsSheetViewEffect? = null
+) : MavericksState {
 
-    /**
-     * Restores existing persisted fields into the current [FinancialConnectionsSheetState]
-     */
-    internal fun from(savedStateHandle: SavedStateHandle): FinancialConnectionsSheetState {
-        return copy(
-            manifest = savedStateHandle.get(KEY_MANIFEST) ?: manifest,
-            authFlowActive = savedStateHandle.get(KEY_AUTHFLOW_ACTIVE) ?: authFlowActive,
-        )
-    }
```

Comment:
There's no need to handle savedStateHandle with the [@PersistState](https://airbnb.io/mavericks/#/persist-state) annotation.